UserSettings/Layouts/default-2021.dwlt

```diff
@@ -14,16 +14,16 @@ MonoBehaviour:
   m_EditorClassIdentifier: 
```

Comment:
Methinks you did not generate the .gitignore correctly. This definitely should not have been included in the initial push and should not be included in the future.